### DISCUSSION TITLE
How to get line & column from span?

### DISCUSSION THREAD

**Discussion by playXE** [swc-project/swc#1969] [Q&A] [answered]
Read topic. I need this to get proper stacktrace debug info in starlight, JS engine :)

**Comment by kdy1** [comment, marked as answer]
See https://docs.rs/swc_common/0.11.1/swc_common/source_map/struct.SourceMap.html

[`lookup_char_pos`](https://docs.rs/swc_common/0.11.1/swc_common/source_map/struct.SourceMap.html#method.lookup_char_pos) can be used to do it.

**Reply by playXE** [reply]
Thanks!
